app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,25 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @posts = Post.where(author_id: params[:user_id])
+    @user = User.find(params[:user_id])
+  end
 
-  def show; end
+  def show
+    @post = Post.find(params[:id])
+    @comments = @post.comments
+    @like = Like.new
+  end
+
+  def new
+    @user = current_user
+    @post = Post.new
+  end
+
+  def create
+    @user = current_user
+    @post = Post.new(author_id: @user, title: params[:post][:title], text: params[:post][:text])
+    @post.author_id = @user.id
+    @post.save
+    redirect_to user_posts_path(@user)
+  end
```

Comment:
Great job so far @asifaminisonline 
**OPTIONAL**

- [ ] Please create a [Private Params Method](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md) to secure your user form inputs.